main.go

```diff
@@ -71,12 +73,69 @@ func main() {
 	defer machine.Close(context.Background())
 
 	runTests(machine)
+
+	// remove old images before uploading new ones
+	removeAllImages()
+
 	cmd := exec.Command("python3", "plot.py")
 	err = cmd.Run()
 	if err != nil {
 		logger.Error(err)
 		return
 	}
+
+	// upload all new images
+	uploadAllImages()
+}
+
+// remove each saved image from previous run
+func removeAllImages() {
+	var errs error
+	err := os.Remove("./savedImages/sensor_ms_pos.jpg")
+	errs = multierr.Combine(errs, err)
+	err = os.Remove("./savedImages/sensor_ms_vels.jpg")
+	errs = multierr.Combine(errs, err)
+	err = os.Remove("./savedImages/sensor_spin_degs.jpg")
+	errs = multierr.Combine(errs, err)
+	err = os.Remove("./savedImages/sensor_sv_vels.jpg")
+	errs = multierr.Combine(errs, err)
+	err = os.Remove("./savedImages/wheeled_ms_pos.jpg")
+	errs = multierr.Combine(errs, err)
+	err = os.Remove("./savedImages/wheeled_ms_vels.jpg")
+	errs = multierr.Combine(errs, err)
+	err = os.Remove("./savedImages/wheeled_spin_degs.jpg")
+	errs = multierr.Combine(errs, err)
+	err = os.Remove("./savedImages/wheeled_sv_vels.jpg")
+	errs = multierr.Combine(errs, err)
+	err = os.Remove("./savedImages/encoded_go_for_rpm.jpg")
+	errs = multierr.Combine(errs, err)
+	err = os.Remove("./savedImages/encoded_go_for_pos.jpg")
+	errs = multierr.Combine(errs, err)
+	err = os.Remove("./savedImages/encoded_go_to_rpm.jpg")
+	errs = multierr.Combine(errs, err)
+	err = os.Remove("./savedImages/encoded_go_to_pos.jpg")
+	errs = multierr.Combine(errs, err)
+	err = os.Remove("./savedImages/encoded_set_rpm_rpm.jpg")
+	errs = multierr.Combine(errs, err)
+	err = os.Remove("./savedImages/controlled_go_for_rpm.jpg")
+	errs = multierr.Combine(errs, err)
+	err = os.Remove("./savedImages/controlled_go_for_pos.jpg")
+	errs = multierr.Combine(errs, err)
+	err = os.Remove("./savedImages/controlled_go_to_rpm.jpg")
+	errs = multierr.Combine(errs, err)
+	err = os.Remove("./savedImages/controlled_go_to_pos.jpg")
+	errs = multierr.Combine(errs, err)
+	err = os.Remove("./savedImages/controlled_set_rpm_rpm.jpg")
+	errs = multierr.Combine(errs, err)
+	err = os.Remove("./savedImages/grid_test.jpg")
+	errs = multierr.Combine(errs, err)
```

Comment:
I think this should work? Other thought is deleting images after they get uploaded/as a defer function in `uploadFiles`

```suggestion
        errs := multierr.Combine(
        os.Remove("./savedImages/sensor_ms_pos.jpg"), 
        os.Remove("./savedImages/sensor_ms_vels.jpg"),
        os.Remove("./savedImages/sensor_spin_degs.jpg"),
        os.Remove("./savedImages/sensor_sv_vels.jpg"),
        os.Remove("./savedImages/wheeled_ms_pos.jpg"),
        os.Remove("./savedImages/wheeled_ms_vels.jpg"),
        os.Remove("./savedImages/wheeled_spin_degs.jpg"),
        os.Remove("./savedImages/wheeled_sv_vels.jpg"),
        os.Remove("./savedImages/encoded_go_for_rpm.jpg"),
        os.Remove("./savedImages/encoded_go_for_pos.jpg"),
        os.Remove("./savedImages/encoded_go_to_rpm.jpg"),
        os.Remove("./savedImages/encoded_go_to_pos.jpg"),
        os.Remove("./savedImages/encoded_set_rpm_rpm.jpg"),
        os.Remove("./savedImages/controlled_go_for_pos.jpg"),
        os.Remove("./savedImages/controlled_go_to_rpm.jpg"),
        os.Remove("./savedImages/controlled_go_to_pos.jpg"),
        os.Remove("./savedImages/controlled_set_rpm_rpm.jpg"),
        os.Remove("./savedImages/grid_test.jpg"))
```

Reply:
my thought was to leave them until the beginning of the next run in case there are issues with an upload or something and the next day someone wants to view them locally